.ansible-lint.yml

```diff
@@ -0,0 +1,8 @@
+---
+skip_list:
+  - role-name
+
+exclude_paths:
+  - vendor/**
+  - ansible/roles/windows-*/**
```

Comment:
We don't have this path in our repo, so lets remove. I think we can keep `vendor/**` because that is a common enough pattern.